README.md

```diff
@@ -894,35 +903,31 @@ template <class D, class P>
 class TorqueGeneratedStruct : public P {
  public:
 ```
-Where D is Struct and P is HeapObject in this case. But the above is the declartion
+Where D is Struct and P is HeapObject in this case. But the above is the declaration
 of the type but what we have in the .h file is what was generated. 
 
 This type is defined in `src/objects/struct.tq`:
 ```
-@abstract                                                                       
-@generatePrint                                                                  
-@generateCppClass                                                               
-extern class Struct extends HeapObject {                                        
-} 
+@abstract
```

Comment:
Refer to https://github.com/v8/v8/blob/ee74e718339223345b42935e9f4cbc7be208e557/src/objects/struct.tq#L5-L7